### PR TITLE
Ap 2914 emp integration tests

### PR DESCRIPTION
**Pull request by willc-work**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/AP-2914)

Add client_id to both the employment and employment_payments sections of the CFE results
Check remarks actual vs expected

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing: `bundle exec rake`
- Github should not be reporting conflicts; you should have recently run `git rebase main`.
- There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- The PR description should say what you changed and why, with a link to the JIRA story.
- You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- You should have checked that the commit messages say why the change was made.

### CODE REVIEW

**Review by stephenrichards — APPROVED** [review]

File: lib/integration_helpers/test_case/v4/expected_result.rb [hunk 1]
```diff
@@ -24,6 +24,27 @@ def populate
         end
       end
 
+      def remarks(rows)
```
Comment (chris-groves) [inline lib/integration_helpers/test_case/v4/expected_result.rb:27]: could this method be refactored?

File: lib/integration_helpers/test_case/v4/expected_result.rb [hunk 2]
```diff
@@ -24,6 +24,27 @@ def populate
...
+          if issue.present?
+            current_issue = issue.to_sym
+            hash[current_type][current_issue] = []
+          end
+
+          hash[current_type][current_issue] << id
+        end
+        @result_set[:remarks] = hash
+      end
+
```
Comment (chris-groves) [inline lib/integration_helpers/test_case/v4/expected_result.rb:47]: there's come commented out code on line 82

File: lib/integration_helpers/test_case/v4/remarks_comparer.rb [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+        @result
+      end
+
+      def silent?
+        @verbosity.zero?
+      end
+
+      def compare_remarks
+        puts "Remarks >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>".green unless silent?
```
Comment (chris-groves) [inline lib/integration_helpers/test_case/v4/remarks_comparer.rb:29]: is this meant to be left here?

File: db/schema.rb [hunk 4]
```diff
@@ -22,8 +21,8 @@
     t.string "involvement_type"
     t.boolean "has_partner_opponent"
     t.boolean "receives_qualifying_benefit"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+    t.datetime "created_at", precision: nil, null: false
```
Comment (chris-groves) [inline db/schema.rb:24]: @stephenrichards these schema changes look similar to the ones from my rubocop PR

**Pull request merged**